app-text/texlive/texlive-2021.recipe

```diff
@@ -384,77 +453,112 @@ PROVIDES="
 	cmd:vpl2ovp
 	cmd:vpl2vpl
 	cmd:vptovf
+	cmd:webquiz
 	cmd:weave
 	cmd:wofm2opl
 	cmd:wopl2ofm
+	cmd:wordcount
 	cmd:wovf2ovp
 	cmd:wovp2ovf
 	cmd:xdvipdfmx
 	cmd:xelatex
-	cmd:xelollipop
+	cmd:xelatex_dev
 	cmd:xetex
+	cmd:xhlatex
+	cmd:xindex
+	cmd:xml2pmx
 	cmd:xmltex
+	cmd:yplan
 	lib:libkpathsea$secondaryArchSuffix
 	lib:libptexenc$secondaryArchSuffix
+	lib:libsynctex$secondaryArchSuffix
+	lib:libtexlua53$secondaryArchSuffix
+	lib:libtexluajit$secondaryArchSuffix
```

Comment:
Maybe add $libversion + $libVersionCompat here?
For libkpathsea it should be 6.3.3 (compat 6)
For libptexenc it should be 1.3.9 (compat 1)
For libtexlua53 it should be 5.3.6 (compat 3)
For libtexluajit and libsynctex it should be 2.1.0 (compat 2)

Reply:
I will add these later (still have work to do for the day job first ...)

I think I can't really use variables for that because the values are nearly all different?

Reply:
A bit more work, but you could look at the next for an example :) 

https://github.com/haikuports/haikuports/blob/14c2cab5428145b93232cb69683a67bbe68a9f06/app-text/poppler/poppler-20.11.0.recipe#L232